### DISCUSSION TITLE
Compact Schedule View

### DISCUSSION THREAD

**Discussion by theClarkSell** [ThatConference/that.us#775] [Ideas]
What would make the best compact view? What would you like to see?

**Comment by petronivs** [comment]
I think title and presenter name are most vital.  
Location would help for quick reference at-con.

**Reply by PrairieGrl** [reply]
Location would be awesome for conference time.

**Comment by PrairieGrl** [comment]
I think an accordion by date or table by date would be awesome. I'd like to be able to glance at what's available on a certain date without a lot of scrolling.

**Comment by TheTopher** [comment]
Two answers (but most likely the same solution)
1) Before conference: Date/time, title, speaker.  Location is a nice to have, but you may not know until conference-time.  Used on desktop/laptop.
2) At conference - Date/time, title, location.  Speaker is a nice to have.  Sortable.  Used for quick reference of selected presentations.   Used on mobile.

**Reply by theClarkSell** [reply]
Well, you kinda need to know the location since we have AT THAT || ON THAT, but I follow. You want some clutter removed when it's not important.

**Reply by TheTopher** [reply]
That makes sense.  I was thinking that "location" would be room number, which you may not know yet.  From the perspective of ON/AT, I think it is important.

**Comment by theClarkSell** [comment]
I was surprised no one mentioned the description. This year we asked speakers to give us a short and a long description and although we got some flack for it, my intent was to be able to use a short description in places like this. 

Is that not something anyone cares about in this view? Would you simply judge based on title and then expect to click on it, to take you to the details of it? and by take you, I would redirect to the full details view vs trying to expand the view.

**Reply by petronivs** [reply]
I thought about the description, but, if you're trying for ultra compact, you want to include as few items of information as possible, while still giving some identifying information about the sessions.  Honestly, the title really should be the 'ultra short description'--it should give some indication of what the session is about.  
I see this as the at-con schedule that people are going to flip through on their cell phone, to find their next session and the room it's in.  They've probably already perused the descriptions back at their hotel room or at breakfast.  
I think the short description should be the one visible in the regular schedule listing, and the long description should be in the 'detail view' where you have all the info on the session, like the presenter's favorite color and the names of their pets.

**Reply by PrairieGrl** [reply]
100% agree, the super compact I only expect to use when I'm at the conference or checking a time quick.
